lib/user.js

```diff
@@ -134,27 +134,33 @@ export class UserClient extends Client {
                 return data.aurls;
             })
             .then(urls => {
+                const renameMap = {
+                    full: 'body',
+                };
                 const upload = (image, url) => {
-                    fetch(url, {
+                    return fetch(url, {
                         method: 'put',
                         body: image,
                         headers: {
                             'Content-Type': 'image/png',
                             'Content-Length': image.size.toString(),
                         }
                     })
-                        .catch(error => {
-                            console.log(error);
-                        });
+                        .then(() => true)
+                        .catch(() => false);
```

Comment:
Shouldn't this let the error happen?

Reply:
As in log the error? We weren't doing anything with the error before